### PR TITLE
[Brl.Math] Add Deg2Rad and Rad2Deg commands

### LINKED ISSUE #344: [Brl.Math] Add Rad2Deg, Deg2Rad
Heya,

while scanning the source codes some syntaxbomb user posted I saw them using a custom "Deg2Rad" function they had to add to the samples.

Brl.Math already contains some prepared variables in bbMath.: `DEG_TO_RAD`, `RAD_TO_DEG` (and their float counterparts).

Maybe exposing that functionality isn't the worst idea.

### PR DESCRIPTION
**Pull request by GWRon**
Closes #344

### CODE REVIEW

File: math.mod/math.c [hunk 1]
```diff
@@ -131,6 +141,8 @@ double bbTrunc( double x ){
 #define ceilf ceil
 #define roundf round
 #define truncf trunc
+#define deg2radf deg2rad
+#define rad2degf rad2deg
```
Comment (woollybah) [inline math.mod/math.c:145]: This is not required.

Comment (GWRon) [reply math.mod/math.c:145]: Can you tell why? 

Or better: why is it needed for the other entries? I simply tried to do as done for all the other functions.

Comment (GWRon) [reply math.mod/math.c:145]: What I mean is - if I remove these lines, should I remove the other ones too?
```

#define sqrtf sqrt
#define sinf sin
#define cosf cos
#define tanf tan
#define asinf asin
#define acosf acos
#define atanf atan
#define atan2f atan2
#define sinhf sinh
#define coshf cosh
#define tanhf tanh
#define expf exp
#define floorf floor
#define logf log
#define log10f log10
#define ceilf ceil
#define roundf round
#define truncf trunc
#define deg2radf deg2rad
#define rad2degf rad2deg
```

Comment (GWRon) [reply math.mod/math.c]: I removed the blamed lines - just drop a line if I should remove the other define-lines (see above) too.

If the PR is then OK, do not forget to "squash" the commits (to remove the extra commit from git history).

**Pull request merged**